workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.ts

```diff
@@ -154,7 +164,7 @@ export class DynamicAuthorizationService {
    * @returns - {@link AddUserToGroupResponse}
    */
   public async addUserToGroup(addUserToGroupRequest: AddUserToGroupRequest): Promise<AddUserToGroupResponse> {
-    throw new Error('Not implemented');
+    return this._groupManagementPlugin.addUserToGroup(addUserToGroupRequest);
```

Comment:
I believe we need to check if `groupStatus === 'active'` to make sure we can add the user to the group. @htztomic can you confirm?